en/06_Project_Governance/06_Supported_Modules.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Supported modules
+---
+
+# Commercially supported modules for Silverstripe CMS 5
+
+Silverstripe CMS ships modules that receive commercial support. Commercially supported modules receive regular updates to work with the latest Silverstripe CMS release. Their APIs conform to semantic version. They are covered by:
+- our [security release process](./Managing_Security_Issues) and 
+- our [major release policy](./Major_release_policy).
+
+These modules will be supported for the lifetime of Silverstripe CMS 5 in the provided major versions.
+
+## Core Silverstripe CMS modules
+
+These modules provide the core Silverstripe CMS experience. Most Silverstripe CMS projects should install all of them.
+
+Core Module | Supported versions
+-- | --
+[silverstripe/admin](https://packagist.org/packages/silverstripe/admin) | 2
+[silverstripe/asset-admin](https://packagist.org/packages/silverstripe/asset-admin) | 2
+[silverstripe/assets](https://packagist.org/packages/silverstripe/assets) | 2
+[silverstripe/cms](https://packagist.org/packages/silverstripe/cms) | 5
+[silverstripe/config](https://github.com/silverstripe/silverstripe-config) | 2
+[silverstripe/campaign-admin](https://packagist.org/packages/silverstripe/campaign-admin) | 2
+[silverstripe/errorpage](https://packagist.org/packages/silverstripe/errorpage) | 2
+[silverstripe/framework](https://packagist.org/packages/silverstripe/framework) | 5
+[silverstripe/graphql](https://packagist.org/packages/silverstripe/graphql) | 5
+[silverstripe/installer](https://github.com/silverstripe/silverstripe-installer) | 5
+[silverstripe/login-forms](https://packagist.org/packages/silverstripe/login-forms) | 5
+[silverstripe/mimevalidator](https://packagist.org/packages/silverstripe/mimevalidator) | 3
+[silverstripe/recipe-cms](https://github.com/silverstripe/recipe-cms) | 5
+[silverstripe/recipe-core](https://github.com/silverstripe/recipe-core) | 5
+[silverstripe/recipe-plugin](https://github.com/silverstripe/recipe-plugin) | 2
```

Comment:
recipe-plugin and vendor-plugin aren't modules.... though I don't know if we need to make this distinction. We probably should since we made a distinction for the npm packages.
It might even make sense to distinguish recipes - though recipes are much more similar to modules than composer plugins are so I'm okay with keeping those lumped in if that's what you prefer.

Reply:
Honestly, I think it's a distinction without a difference ... and it saves me drafting a dissertation about the difference between a module, a package and a plugin.

Reply:
For the composer plugins it's almost as much of a difference as the npm packages which you've separated out - but I'm fine with leaving these as is, it's not a big deal. Just my brain being pedantic.